ee/maintained-apps/ingesters/homebrew/scripts.go

```diff
@@ -326,6 +330,10 @@ sudo installer -pkg "$TMPDIR"/%s -target / -applyChoiceChangesXML "$CHOICE_XML"
 // Symlink writes a command to create a symbolic link from 'source' to 'target'.
 func (s *scriptBuilder) Symlink(source, target string) {
 	pathname := filepath.Dir(target)
+	if _, ok := s.pathsCreated[pathname]; !ok {
+		s.Writef("mkdir -p %s", pathname)
+		s.pathsCreated[pathname] = struct{}{}
+	}
 	s.Writef(`[ -d "%s" ] && /bin/ln -h -f -s -- "%s" "%s"`, pathname, source, target)
```

Comment:
```suggestion
	s.Writef(`/bin/ln -h -f -s -- "%s" "%s"`, pathname, source, target)
```

@jahzielv now that we make the dir ahead of time, we can indeed drop this, based on looking back at the last time this was changed, in #25901.